lib/ui/form/widget/form_survey_answer_nps.dart

```diff
@@ -22,12 +25,23 @@ class FormSurveyAnswerNps extends ConsumerStatefulWidget {
 class FormSurveyAnswerNpsState extends ConsumerState<FormSurveyAnswerNps> {
   final selectedScoreProvider = StateProvider.autoDispose<int>((_) => 0);
   late int scoresLength;
+  late List<String> _answerIds;
 
   @override
   void initState() {
     super.initState();
     scoresLength = widget.question.answers.length;
-    ref.read(selectedScoreProvider.notifier).state = (scoresLength / 2).floor();
+    final int defaultSelectedAnswer = (scoresLength / 2).floor();
+    ref.read(selectedScoreProvider.notifier).state = defaultSelectedAnswer;
+    _answerIds = widget.question.answers.map((element) => element.id).toList();
+    widget.onUpdateAnswer(
+      [
+        SubmitAnswerModel(
+          answerId: _answerIds[defaultSelectedAnswer],
+          answerText: null,
```

Comment:
Can we set it as default value from `SubmitAnswerModel` and get rid of `answerText: null,` set here?